docs/glossary.asciidoc

```diff
@@ -0,0 +1,32 @@
+[id="{p}-glossary"]
+== Glossary
+
+This glossary covers Elastic Cloud on Kubernetes. If a term is not in this list, you can also look in the link:https://kubernetes.io/docs/reference/glossary/[Kubernetes glossary].
+
+[[GKE]]GKE::
+  Google Kubernetes Engine (GKE): is a managed, production-ready environment for deploying containerized applications.
+
+[[AKS]]AKS::
```

Comment:
nit: I don't think AKS is referenced in the documentation (there's just a reference to Azure in `elasticsearch-spec.asciidoc`)